packages/helper/src/test/ts/helper/helper.ts

```diff
@@ -1,24 +1,36 @@
 import { ComponentsApi, SearchApi } from '@qiwi/nexus-client'
 import nock from 'nock'
 
+import { TComponent } from '../../../main/ts'
 import { NexusComponentsHelper } from '../../../main/ts/helper'
 import component from './resources/component.json'
 
 describe('NexusContentsHelper', () => {
   const basePath = 'http://localhost/service/rest'
   const componentsApi = new ComponentsApi({ basePath })
   const searchApi = new SearchApi({ basePath })
-  const helper = new NexusComponentsHelper(searchApi, componentsApi)
+  const wrapperOpts = {
+    delay: {
+      period: 100,
+      count: 4,
+    }
```

Comment:
This just might be mentioned in docs as recommended value

Reply:
I think it's not good idea, because I came up with these values